docs/contributor/sitemap-generator/README.md

```diff
@@ -66,6 +66,11 @@ pip3 install -e .
 Note: Ensure the location you install sitemap-generator is on your `$PATH`, otherwise you
 won't be able to use the sitmap-generator CLI.
 
+```bash
+echo $PATH
+find $HOME -name sitemap-generator -type f
```

Comment:
Can you add comment on previous line that the command searchs for location of where pip3/python put it? Note this only applies to local/user install and shouldn't be necessary with global install.